src/game/start.py

```diff
@@ -90,10 +119,19 @@ def selected_player_to_string(player_name, selected):
     return player_name
 
 
+def cls():
```

Comment:
The function name isn't explicit enough. `clear_terminal` or something would be better IMO